spec/rails_helper.rb

```diff
@@ -51,4 +59,19 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+ # def login(email, password)
+ #   visit login_path
+ #   fill_in :email, with: email
+ #   fill_in :password, with: password
+ #   click_button "Login"
+ # end
+ module Sorcery
+ module TestHelpers
+  module Rails
+      def login_user_post(user, password)
+        page.driver.post(user_sessions_url, { email: user.email, password: password}) 
```

Comment:
Line is too long. [86/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.<br>Trailing whitespace detected.